wqflask/wqflask/templates/correlation_matrix.html

```diff
@@ -26,35 +27,53 @@
       <td align="center" rowspan="{{traits|length + 1 }}" width="10" style="font-weight: Bold; border: 1px solid #999999; padding: 3px; color: #fff; background-color: royalblue;">P e a r s o n &nbsp;&nbsp;&nbsp; r</td>
```

Comment:
Can we start using CSS for these HTML attributes? This should really be a css class!